conf/app.go

```diff
@@ -0,0 +1,5 @@
+package conf
```

Comment:
Can I use this file for make a database configuration? or you have another file else

Reply:
sure, why not...